src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.2.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>2.0.0</RoslynSemanticVersion>
```

Comment:
Is it possible to leave this unless we actually make changes?  I'd actually like to see it stay at 1.1.0.